analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ToStringShouldNotReturnNull.CSharp11.cs

```diff
@@ -2,7 +2,16 @@
 {
     static virtual string ToString()
```

Comment:
Please also add a test case with a static ToString method in a class that does not come from an interface. (Add that to the base test cases)